src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -326,11 +326,12 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             }
 
             // Index expressions
-            if (previousKind == SyntaxKind.CaretToken && previousParentKind == SyntaxKind.IndexExpression)
+            if (previousKind == SyntaxKind.CaretToken && previousParentKind == SyntaxKindEx.IndexExpression)
             {
                 return CreateAdjustSpacesOperation(0, AdjustSpacesOption.ForceSpaces);
             }
 
+#if !CODE_STYLE
             // Right of Range expressions
             if (previousKind == SyntaxKind.DotDotToken && previousParentKind == SyntaxKind.RangeExpression)
```

Comment:
i'd love a strategy on how we avoid whatever this is (or track removing it as soon as we can).  Given this is a range-expression, i assume this is something to do with new language feature and CodeStyle vs. Workspaces referencing different versions of the compiler.  Seems ok to have this, but i think it should be associated with a bug that tracks removing it.

Reply:
Versioning strategy for analyzers distributed via NuGet is an open discussion.